index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
Emotes can cause problems in some systems.
Could you:
Find an alternative
Remove the emotes or 
Assure this will not cause issue with our system.
Any of the three will do